pkg/server/update_runtime_config.go

```diff
@@ -49,10 +50,21 @@ func (c *criService) UpdateRuntimeConfig(ctx context.Context, r *runtime.UpdateR
 		logrus.Info("No cni config template is specified, wait for other system components to drop the config.")
 		return &runtime.UpdateRuntimeConfigResponse{}, nil
 	}
+	var copts []cni.CNIOpt
+	if osruntime.GOOS != "windows" {
+		copts = append(copts, cni.WithLoNetwork)
+	}
+	copts = append(copts, cni.WithDefaultConf)
+
+	// Load the latest cni configuration to be in sync with the latest network configuration
+	if err := c.netPlugin.Load(copts...); err != nil {
```

Comment:
You dont want to call Load here right? You just want to make the opts and then wait for the "else if" to trigger the load?

Reply:
Yes, Let me remove it